blog/2022/08-30-discover-kubernetes/index.md

```diff
@@ -0,0 +1,303 @@
+---
+authors: [matthias]
+tags: [cloud, search, graph, k8s]
+image: ./img/banner-social.png
+---
+
+# Discover Kubernetes Resources
+
+Kubernetes has dramatically improved the way we manage our workloads. It has become the de facto standard for deploying and managing containers.
+
+A typical setup uses different Kubernetes clusters for different application stages (dev, test, prod) or one cluster per tenant. A Kubernetes cluster shared between different users and teams usually uses namespaces and roles to control access. Deploying a single application to a Kubernetes cluster usually consists of 10s to 100s of resources to deploy (deployments, services, configmaps, secrets, ingresses, etc.).
+
+What starts simple in the beginning becomes tedious quite fast since the number of resources is vast and the view of a single user is limited.
```

Comment:
can you explain why specifically the view for the individual user is limited, and why that is a problem?